docs/reference/search/search-your-data/retrievers-overview.asciidoc

```diff
@@ -0,0 +1,211 @@
+allowing you to build pipelines and seamlessly integrate different retrieval 
+strategies into these pipelines. Retrievers make it easy to test out different 
+retrieval strategy combinations.
+. *Compound operations*. A retriever can have sub retrievers. This 
+allows complex nested searches where the results of one retriever feed into 
+another, supporting sophisticated querying strategies that might involve 
+multiple stages or criteria.
+. *Retrieval as a first-class concept*. Unlike 
+traditional queries, where the query is a part of a larger search API call, 
+retrievers are designed as standalone entities that can be combined or used in 
+isolation. This enables a more modular and flexible approach to constructing 
+searches.
+. *Enhanced control over document scoring and ranking*. Retrievers 
+allow for more explicit control over how documents are scored and filtered. For 
+instance, you can specify minimum score thresholds, apply complex filters 
+without affecting scoring, and use parameters like `terminate_after` for 
+performance optimizations.
+. *Integration with existing {es} functionalities*. Even though 
+retrievers can be used instead of existing `_search` API syntax (like the 
+`query` and `knn`), they are designed to integrate seamlessly with things like
+pagination (`search_after`) and sorting. They also maintain compatibility with 
+aggregation operations by treating the combination of all leaf retrievers as 
+`should` clauses in a boolean query.
+. *Cleaner separation of concerns*. When using compound retrievers, only the 
+query element is allowed, which enforces a cleaner separation of concerns 
+and prevents the complexity that might arise from overly nested or 
+interdependent configurations.
+
+[discrete]
+[[retrievers-overview-example]]
+=== Example
+
+The following example demonstrates how using retrievers 
+simplify the composability of queries for RRF ranking.
+
+[source,js]
+----
+GET example-index/_search
+{
+  "retriever": {
+    "rrf": {
+      "retrievers": [
+        {
+          "standard": {
+            "query": {
+              "text_expansion": {
+                "vector.tokens": {
+                  "model_id": ".elser_model_2",
+                  "model_text": "What blue shoes are on sale?"
+                }
+              }
+            }
+          }
+        },
+        {
+          "standard": {
+            "query": {
+              "multi_match": {
+                "query": "blue shoes sale",
+                "fields": [
+                  "text"
+                ]
+              }
+            }
+          }
+        }
+      ]
+    }
+  }
+}
+----
+//NOTCONSOLE
+
+This example demonstrates how you can combine different
+retrieval strategies into a single `retriever` pipeline.
+
+Compare to `RRF` with `sub_searches` approach:
+
+.*Expand* for example
+[%collapsible]
+==============
+
+[source,js]
+----
+GET example-index/_search
+{
+    "sub_searches": [
```

Comment:
Nit: consider using the same indentation as in the previous example (2 vs 4 spaces).